addons/rose/addon/components/rose/filter-tags/index.js

```diff
@@ -0,0 +1,81 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+export default class RoseFilterTagsIndexComponent extends Component {
+  // =services
+
+  @service router;
+
+  // =attributes
+  get tags() {
+    const tags = [];
+
+    for (const key in this.args.tags) {
+      if (Array.isArray(this.args.tags[key])) {
+        const names = this.args.tags[key].map((item) => item.name);
+        tags.push(...names);
```

Comment:
This works but we also could have just done `const tags = flatMap(...)` for this

Reply:
Nice. That is cleaner.